core/cli/src/lib/utils/procesmanager/TaskWorker.ts

```diff
@@ -88,7 +88,7 @@ export class TaskWorker {
 
   #debounceClean: NodeJS.Timeout | undefined;
   #clean = ({ source, msg }: { source: string; msg: any }) => {
-    if (!!!msg) {
+    if (!msg) {
       /** no message means a clean exit from the worker, don't restart */
```

Comment:
Could you please tell me why you removed the explicit cast to boolean? I recall I put it there for an edge case, but I can't recall the case tho.

Reply:
In the first lint run I think it was removed, there everything was realllly strict / explicit. But I added it back